README.md

```diff
@@ -108,3 +108,17 @@ $this->bus->dispatch((new Envelope($message))->with(new TransportConfiguration([
     ]
 ])));
 ```
+
+### Configure custom Kafka message
+
+Here is the way to send a message with with some custom options:
+```php
+$this->bus->dispatch((new Envelope($message))->with(new TransportConfiguration([
+    'topic' => 'test_topic_name',
+    'metadata' => [
+        'key' => 'foo.bar',
+        'partition' => 0,
+        'timestamp' => (new \DateTimeImmutable())->getTimestamp(),
+        'messageId' => uniqid('kafka_', true),
+    ]
+])))
```

Comment:
I think this is a valid thing to show. However, I want to think through how someone will think about these options. I'm just guessing your mind would go like this (but I could *totally* be wrong, so tell me):

1) You're using Kafka, and you decide you want to add a timestamp
2) You search through Enqueue's Kafka adapter to figure out how. There's nothing in the docs, https://github.com/php-enqueue/enqueue-dev/blob/master/docs/transport/kafka.md, but you eventually find out that there is a setter method on the message object: https://github.com/php-enqueue/enqueue-dev/blob/c58e1af68b49612e597dc80a821444488f1af210/pkg/rdkafka/RdKafkaMessage.php#L142
3) You then try to figure out how you can call that method on the Enqueue message via the transport

If I'm correct, then the header should be something like: `How to set Custom Options onto Enqueue Message Objects` and it should take about how the `metadata` are mapped to the setter methods of the underlying Enqueue message object.

Reply:
#55 might be an example of what I was thinking - user wanted to call a setter on the message object.

Reply:
And #58. So yea, apparently this is quite confusing!

Reply:
Sorry for the noise. I just added #61. Let me know if you still want to add some extra details. If you *do*, it might be best as an other example up in the new section I added.

Cheers!

Reply:
Yes, you got it. I can change it but I want to highlight Kafka because for AMQP there is some info already.

Reply:
So do you really need to have these `timestamp` and `messageId` attributes for Kafka? 🤔

Reply:
It just to highlight some options available to the user.